Add fixes for building rt-tests image

yrobla opened the pull request:
When building the rt-tests images, some failures
    appeared:
    - error handling not done properly. There are cases
    where commands return exit code 1, that are valid,
    but were causing script to exit. Do a proper
    management of those errors.
    - add special case for rt-tests, that do not need
    the building of oai-base, that is long and heavy
    - fix typo in rt-tests Dockerfile

oglok reviewed (commented):
Setting -e as a general rule is a bad practice in bash, while we should be using traps or checking for errors. I don't know if enabling/disabling exit errors is a good idea, or we should just revert the previous merged commit.

Commit:
Add fixes for building rt-tests image

When building the rt-tests images, some failures
appeared:
- error handling not done properly. There are cases
where commands return exit code 1, that are valid,
but were causing script to exit. Do a proper
management of those errors.
- add special case for rt-tests, that do not need
the building of oai-base, that is long and heavy
- fix typo in rt-tests Dockerfile
